template/README.md

```diff
@@ -658,8 +673,6 @@ Create Elm App uses [elm-test](https://github.com/rtfeldman/node-test-runner) as
 
 To use packages in tests, you also need to install them in `tests` directory.
 
-Please note that you have to pass the path to the `elm-package.json` with your test dependencies.
```

Comment:
This line shouldn't be removed, could you please reset this change?

Reply:
Changed it. I'm regretting, that I started with the last release and didn't branch away from master ;)